src/reader-activation-newsletters/style.scss

```
@@ -5,16 +5,11 @@
 	}
 
 	p.details {
-		margin-bottom: 0;
+		margin-bottom: 4px;
```

```suggestion
		margin-bottom: 0;
```

Another little regression -- this was all done in https://github.com/Automattic/newspack-plugin/pull/3273